src/features/common/Layout/Navbar/index.tsx

```diff
@@ -15,72 +14,86 @@ import { ThemeContext } from '../../../../theme/themeContext';
 import styles from './Navbar.module.scss';
 import i18next from '../../../../../i18n';
 import getImageUrl from '../../../../utils/getImageURL'
-import { getUserExistsInDB, getUserInfo } from '../../../../utils/auth0/localStorageUtils'
+import { useAuth0 } from '@auth0/auth0-react';
+import { getUserInfo } from '../../../../utils/auth0/userInfo';
 
 const { useTranslation } = i18next;
 const config = tenantConfig();
 export default function NavbarComponent(props: any) {
-  // If there is a session we will use it
-  const [session, loading] = useSession();
 
   const { t, ready } = useTranslation(['common']);
   const router = useRouter();
 
-  /* Works when user clicks on Me
-   If the user is logged in, redirect to t/userSlug
-   If user is not logged in we will redirect to singin page with Auth0
-   If in the signin flow, if the user is already existing, we login the user and  redirect to t/userSlug
-   If in the signin flow, if the user is not existing, then we redirect to complete Signup flow */
-  const checkWhichPath = () => {
+  const {
+    isLoading,
+    isAuthenticated,
+    error,
+    loginWithRedirect,
+    logout,
+    getAccessTokenSilently
+  } = useAuth0();
 
-    if (typeof Storage !== 'undefined') {
-      
-    const userExistsInDB = getUserExistsInDB();
-  
-    // if user logged in, and already signed up -> /t/userSlug page
-      if (!loading && session && (userExistsInDB === true)) {
-          var userslug = getUserInfo().slug;
-          if (typeof window !== 'undefined') {
-            router.push(`/t/${userslug}`);
-          }
-       // if user logged in, not already signed up -> /complete-signup
-      } else if ( !loading && session && (userExistsInDB === false)) {
-          if (typeof window !== 'undefined') {
-              router.push('/complete-signup');
-          }
-      } else { 
-        // if no user logged in  -> signIn()
-        // or when no active session
-        signIn('auth0', { callbackUrl: `/login` });
-      }
+  const [token, setToken] = React.useState('')
+  const [userInfo, setUserInfo] = React.useState({})
+
+  // This effect is used to get and update UserInfo if the isAuthenticated changes
+  React.useEffect(() => {
+    async function loadFunction() {
+      const token = await getAccessTokenSilently();
+      setToken(token);
+      let userInfo;
+      userInfo = await getUserInfo(token, router, logout);
+      setUserInfo(userInfo);
+    }
+    if (isAuthenticated) {
```

Comment:
If you login and afterwards open https://localhost/ again in your browser `isAuthenticated` is `false` and `isLoading` is still `true` and this code is never called again! So if you try to open the profile afterwards, you get presented the login again. 

(If you login for the first time, this code is executed twice. On the first run also with `isAuthenticated: false` and `isLoading: true`, but on the second run with  `isAuthenticated: true` and `isLoading: false`, so the user info gets loaded.)

Reply:
changed auth0 cacheLocation to localstorage